examples/kanji.py

```diff
@@ -0,0 +1,53 @@
+"""Example for Kanji features."""
+
+from escpos.printer import Usb
+
+checkerboard_kanji = (
+    b"\xf0\xf0\xf0"
+    b"\xf0\xf0\xf0"
+    b"\xf0\xf0\xf0"
+    b"\xf0\xf0\xf0"
+    b"\x0f\x0f\x0f"
+    b"\x0f\x0f\x0f"
+    b"\x0f\x0f\x0f"
+    b"\x0f\x0f\x0f"
+    b"\xf0\xf0\xf0"
+    b"\xf0\xf0\xf0"
+    b"\xf0\xf0\xf0"
+    b"\xf0\xf0\xf0"
+    b"\x0f\x0f\x0f"
+    b"\x0f\x0f\x0f"
+    b"\x0f\x0f\x0f"
+    b"\x0f\x0f\x0f"
+    b"\xf0\xf0\xf0"
+    b"\xf0\xf0\xf0"
+    b"\xf0\xf0\xf0"
+    b"\xf0\xf0\xf0"
+    b"\x0f\x0f\x0f"
+    b"\x0f\x0f\x0f"
+    b"\x0f\x0f\x0f"
+    b"\x0f\x0f\x0f"
+)
+
+p = Usb(0x04B8, 0x0E1F, {}, profile="TM-T20II")
+
+p.set_kanji_encoding("iso2022_jp")
+
+p.set(align="center")
+
+p.set_kanji_decoration(double_height=True)
+p.set_kanji_underline(2)
+p.kanji_text("漢字モード\n")
+p.set_kanji_decoration()
+p.ln()
+
+p.kanji_text("こんにちは世界！\n")
+p.ln()
+
+p.define_user_defined_kanji(b"\x77\x7e", checkerboard_kanji)
+p._enter_kanji_mode()
+p._raw(b"\x77\x7e")
+p._exit_kanji_mode()
```

Comment:
Do I understand this feature correctly, this sequence is used to write the user defined kanji? In this case I would propose an API function such as `write_user_defined_kanji(string)` in order to write the kanji and prevent forcing users to use the "internal" functions.

Reply:
Yes. That seems to be a better way to go. I'll implement them but the input will be hex value.

In some encoding system, some code points are left undefined, and in this example we're writing a new glyph to one of those code point. The undefined zone differs by both encoding and maybe printers, so basically user need to input hex value of code point manually.

Reply:
I agree, a hex value is better for the code point (and aligns with the rest of the code that you wrote).